src/videojs-hls.js

```diff
@@ -3,688 +3,518 @@
  * The main file for the HLS project.
  * License: https://github.com/videojs/videojs-contrib-hls/blob/master/LICENSE
  */
+ /*jshint -W092 */
+ /*jshint -W062 */
```

Comment:
I was adding these overrides because of the jshint error I was getting while grunting the repo. Mixed spaces in the repo.
Do these overrides cause any issues ? Should I remove ?